packages/manager/apps/web/client/app/domain/list/restore-renew/routing.js

```diff
@@ -9,7 +9,9 @@ export default /* @ngInject */ ($stateProvider) => {
     },
     resolve: {
       breadcrumb: () => null,
-      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+      goBack: /* @ngInject */ ($state) => () => {
+        return $state.go('^');
+      },
```

Comment:
```suggestion
      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
```
No need change here. The short way was good.

Reply:
oups my bad, did not take care of this!